src/base/bittorrent/torrenthandle.h

```diff
@@ -399,7 +399,7 @@ namespace BitTorrent
         void handleMetadataReceivedAlert(const libtorrent::metadata_received_alert *p);
         void handleStatsAlert(const libtorrent::stats_alert *p);
 
-        void resume_impl(bool forced, bool uploadMode);
+        void resume_impl(bool forced);
```

Comment:
is it sensible to remove this and call `resume(bool forced)` directly?

Reply:
It is generally a good idea to separate the implementation and the public entry point when implementation is used in multiple places. So I'd better leave it as is.